llama_stack/providers/utils/memory/openai_vector_store_mixin.py

```diff
@@ -36,13 +37,20 @@
     VectorStoreSearchResponse,
     VectorStoreSearchResponsePage,
 )
+from llama_stack.providers.utils.kvstore.api import KVStore
 from llama_stack.providers.utils.memory.vector_store import content_from_data_and_mime_type, make_overlapped_chunks
 
 logger = logging.getLogger(__name__)
 
 # Constants for OpenAI vector stores
 CHUNK_MULTIPLIER = 5
 
+VERSION = "v3"
```

Comment:
Curious - why v3 in here? I thought this is faiss specific, so left is out of qdrant changes?